server/src/configs/server.js

```diff
@@ -1,42 +1,44 @@
 import express from 'express';
+import mongoose from 'mongoose';
 import env from './env';
 import routes from '../routes';
 
 const app = () => {
-    let application;
-    const server = express();
+  let application;
+  const server = express();
 
-    const create = () => {
-        server.set('port', env.port);
-        server.use(express.json());
-        server.use(express.urlencoded({ extended: false }));
-        routes.init(server);
-    };
+  const create = () => {
+    server.set('port', env.port);
+    server.use(express.json());
+    server.use(express.urlencoded({ extended: false }));
+    routes.init(server);
+  };
 
-    const start = () => {
-        create();
-        const port = server.get('port');
-        application = server.listen(port, () => {
-            console.log(`Tripen server active on port ${port}`);
-        });
-    };
+  const start = () => {
+    create();
+    mongoose.connect(env.databaseURI, { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true });
```

Comment:
mongoose.connect returns a promise, use async/await and also handle the cases in which connection is failed, you can add this in separate PR and then merge that PR and rebase this to new master and push again. Or you can do the changes in this PR.